html/includes/authentication/ldap.inc.php

```diff
@@ -108,8 +108,9 @@ function get_userlevel($username)
     // Loop the list and find the highest level
     foreach ($entries as $entry) {
         $groupname = $entry['cn'][0];
+        $userlevel = array();
```

Comment:
won't this break other uses of get_userlevel()?

Perhaps, `return max($userlevel);` below?